telco_cable_colors.py

```diff
@@ -0,0 +1,7 @@
+import telco_cable_pairs_utility
+if __name__ == '__main__':

```

Comment:
This functionality does not get tested with the others. any possibilities to test it?